### PR TITLE
chore: add skeleton variables description

### PR DESCRIPTION
**Pull request by zhpenkov**
Adds variables description to the Skeleton component.

### CODE REVIEW

**Review by yordan-mitev — APPROVED** [review]

File: packages/bootstrap/scss/skeleton/_variables.scss [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
+/// The transform scale of the Skeleton text.
+/// @group skeleton
 $kendo-skeleton-text-transform: scale( 1, .6 ) !default;
+/// The border radius of the Skeleton text.
+/// @group skeleton
 $kendo-skeleton-text-border-radius: $kendo-border-radius-md !default;
 
-$kendo-skeleton-rect-border-radius: 0 !default;
 
+/// The border radius of the rectangle Skeleton.
```
Comment (epetrow) [inline packages/bootstrap/scss/skeleton/_variables.scss:11]: Isn't rectangular Skeleton better here?

File: packages/bootstrap/scss/skeleton/_variables.scss [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+/// @group skeleton
+$kendo-skeleton-rect-border-radius: 0 !default;
+/// The border radius of the circle Skeleton.
```
Comment (epetrow) [inline packages/bootstrap/scss/skeleton/_variables.scss:14]: Same as above, circular instead of circle?

Comment (epetrow) [reply packages/bootstrap/scss/skeleton/_variables.scss:14]: This applies to the other themes as well, also maybe we should separate the two variables with a newline for consistency?

**Review by epetrow — APPROVED** [review]
lgtm

**Pull request merged**